modules/layers/src/column-layer/column-layer.js

```diff
@@ -89,14 +99,19 @@ export default class ColumnLayer extends Layer {
 
   updateState({props, oldProps, changeFlags}) {
     super.updateState({props, oldProps, changeFlags});
+
     const regenerateModels =
-      props.fp64 !== oldProps.fp64 || props.diskResolution !== oldProps.diskResolution;
+      props.fp64 !== oldProps.fp64 ||
+      props.diskResolution !== oldProps.diskResolution ||
+      props.filled !== oldProps.filled ||
+      props.wireframe !== oldProps.wireframe;
```

Comment:
After a second thought, based on the documentation, ```Requires the `extruded` prop to be true.``` the wireframe mode should only be enabled when both `wireframe` and `extruded` set to true.

Reply:
Cool, so I should also add `extruded` prop to updateState.

Reply:
The documentation does not sound accurate to me. It was copied from the PolygonLayer which has stroke mode (path with width) when flat and wireframe mode (1-pixel line) when extruded. If you plan to implement an equivalent stroke mode, cool. If not, I prefer to have some sort of outline when flat over nothing.